.github/workflows/unused-deps.yml

```diff
@@ -0,0 +1,34 @@
+on:
+  workflow_dispatch: {}
+  # Once per day at 00:00 UTC
+  push:
+    branches: [main]
+  pull_request:
+
+env:
+  RUSTFLAGS: -D warnings
+
+jobs:
+    unused-deps:
+        runs-on: ubuntu-latest
+        name: unused dependencies
+        steps:
+        - name: Checkout sources
+          uses: actions/checkout@v3
+        - name: Install toolchain
+          uses: dtolnay/rust-toolchain@nightly
+
+        - name: Install cargo-udeps
+          run: cargo install cargo-udeps --locked
+
+        - name: Check for unused dependencies
+          run: cargo +nightly udeps --all-features --all-targets
+
+        - uses: JasonEtco/create-an-issue@v2
+          if: ${{ failure() }}
+          env:
+            GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+            WORKFLOW_URL: ${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}
+          with:
+            update_existing: true
+            filename: .github/SANITY_UNUSED_DEPS_ISSUE_TEMPLATE.md
```

Comment:
Since the job will run in the CI of each PR, this is not needed

Reply:
removed